src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -601,7 +601,8 @@ inlineCommands = M.fromList $
   , ("thanks", note <$> grouped block)
   , ("footnote", note <$> grouped block)
   , ("verb", doverb)
-  , ("lstinline", skipopts *> dolstinline)
+  , ("lstinline", do options <- option [] keyvals
```

Comment:
I tried to move, but I get an compiler error `Variable not in scope: options :: [([Char], String)]`

```diff
diff --git a/src/Text/Pandoc/Readers/LaTeX.hs b/src/Text/Pandoc/Readers/LaTeX.hs
index 2e40e8eb..11d64340 100644
--- a/src/Text/Pandoc/Readers/LaTeX.hs
+++ b/src/Text/Pandoc/Readers/LaTeX.hs
@@ -601,8 +601,7 @@ inlineCommands = M.fromList $
   , ("thanks", note <$> grouped block)
   , ("footnote", note <$> grouped block)
   , ("verb", doverb)
-  , ("lstinline", do options <- option [] keyvals
-                     dolstinline options)
+  , ("lstinline", dolstinline)
   , ("Verb", doverb)
   , ("url", (unescapeURL <$> braced) >>= \url ->
        pure (link url "" (str url)))
@@ -717,11 +716,12 @@ doverb = do
   marker <- anyChar
   code <$> manyTill (satisfy (/='\n')) (char marker)
 
-dolstinline :: PandocMonad m => [(String, String)] -> LP m Inlines
-dolstinline opts = do
+dolstinline :: PandocMonad m => LP m Inlines
+dolstinline = do
+  options <- option [] keyvals
   marker <- char '{' <|> anyChar 
   codeWith ("",classes,[]) <$> manyTill (satisfy (/='\n')) (char '}' <|> char marker)
-               where classes = case lookup "language" opts >>= fromListingsLanguage of
+               where classes = case lookup "language" options >>= fromListingsLanguage of
                                Just l  -> [l]
                                Nothing -> []

```

Other function using `options <- option [] keyvals` have `LP m Blocks` in there signature. Does this matter?